packages/pieces/mailer-lite/src/lib/actions/create-or-update-subscription.ts

```diff
@@ -0,0 +1,53 @@
+import MailerLite from "@mailerlite/mailerlite-nodejs";
+import { createAction, Property } from "@activepieces/framework";
+
+export const createOrUpdateSubscriber = createAction({
+    name: 'add_or_update_subscriber',
+    displayName: "Add or Update subscriber",
+    description: "Create or update a existing subscription",
+    props: {
+        api_key: Property.ShortText({
```

Comment:
Replace with SecretText , this will allow the key to be reusable and appears in app connections.